pkg/oc/cli/admin/release/new.go

```diff
@@ -742,6 +744,14 @@ func (o *NewOptions) Run() error {
 		return err
 	}
 
+	if err := ensureConsistentCommits(is); err != nil {
+		if o.AllowCommitDivergence {
```

Comment:
invert this condition, return early.  Print outside the else.

Reply:
> invert this condition, return early. Print outside the else.

Done with c9d8e9db96 -> 0a1557b5fa.